src/group_theory/specific_groups/quaternion_group.lean

```diff
@@ -7,7 +7,8 @@ import data.zmod.basic
 import group_theory.order_of_element
 import data.nat.basic
 import tactic.interval_cases
-import group_theory.dihedral_group
+import group_theory.specific_groups.dihedral_group
+import group_theory.specific_groups.cyclic_group
```

Comment:
Should we prune the trailing `_group` here?
```suggestion
import group_theory.specific_groups.dihedral
import group_theory.specific_groups.cyclic
```
(obviously after renaming the files)

The word `group` already appears three times in this name!

Reply:
Bryan said that he thought the files should be named after the main definition in the file and the definition `quaternion_group` can't be replaced by `quaternion` since this is already taken.